content/docs/1.7.0/deploy/important-notes/index.md

```diff
@@ -104,161 +62,78 @@ For example, adding the following labels to the namespace that is running Longho
 
-### Replacing `backendStoreDriver` with `dataEngine`
+### Introduction of Filesystem Trim
 
-The attribute `backendStoreDriver`, which is defined in the parameters of StorageClasses and other Longhorn resources (for example, volumes, engines, and replicas), has been replaced with `dataEngine`. You must remove the existing StorageClasses for V2 volumes and create new ones that use `dataEngine`.
+Filesystem trim is supported since Longhorn v1.7.0. If a disk is managed by the SPDK AIO bdev driver, the Trim (UNMAP) operation is not recommended in a production environment (ref). It is recommended to manage a block-type disk with an NVMe bdev driver.
 
-### Updating the Linux Kernel on Longhorn Nodes
+### Linux Kernel on Longhorn Nodes
 
-Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. Update the Linux kernel on Longhorn nodes to version 5.19 or later to prevent such issues. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/).
+Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. To prevent this, update the Linux kernel on Longhorn nodes to version 5.19 or later. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/). Additionally, version 6.7 or later is highly recommended for improved system stability.
 
-### Storage Network Support for Read-Write-Many (RWX) Volumes
+### Snapshot Creation Times Change from Time to Time
 
-Starting with Longhorn v1.7.0, the [storage network](../../advanced-resources/deploy/storage-network/) supports RWX volumes. However, the network's reliance on Multus results in a significant restriction.
+Snapshots created before Longhorn v1.7.0 changes from Time to Time. This issue arises because the engine randomly selects a replica and its snapshot map each time the UI requests snapshot info, leading to potential 1-second gaps between snapshots among different replicas. This bug was fixed in v1.7.0, but snapshots created before this version may still encounter the issue. For more information, see this [link](https://github.com/longhorn/longhorn/issues/7641).
 
-Multus networks operate within the Kubernetes network namespace, so Longhorn can mount NFS endpoints only within the CSI plugin pod container network namespace. Consequently, NFS mount connections to the Share Manager pod become unresponsive when the CSI plugin pod restarts. This occurs because the namespace in which the connection was established is no longer available.
+### Unable to Revert a Volume to a Snapshot Created before Longhorn v1.7.0
+
+Reverting a volume to a snapshot created before Longhorn v1.7.0 is not supported due to an incorrect UserCreated flag set on the snapshot. A workaround is to back up the existing snapshots before upgrading to Longhorn v1.7.0 and restore them if needed. The bug is fixed in v1.7.0, and more information can be found [here](https://github.com/longhorn/longhorn/issues/9054).
```

Comment:
A minor suggestion 'A workaround' suggests there may be more than one. Using 'The workaround' sounds more specific.

```suggestion
Reverting a volume to a snapshot created before Longhorn v1.7.0 is not supported due to an incorrect UserCreated flag set on the snapshot. The workaround is to back up the existing snapshots before upgrading to Longhorn v1.7.0 and restore them if needed. The bug is fixed in v1.7.0, and more information can be found [here](https://github.com/longhorn/longhorn/issues/9054).
```